common/common.go

```diff
@@ -16,6 +16,7 @@ const (
 )
 
 const (
+	DefaultImageDir               = "/var/lib/sealer"
```

Comment:
confused about the name DefaultImageDir。

Reply:
the default image root dir